src/components/TodosTable/TodosTable.tsx

```diff
@@ -0,0 +1,130 @@
+import { useState, useEffect, FC } from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+type Props = {
+  selectedPersonSlug: string;
+};
+
+export const TodosTable: FC<Props> = ({ selectedPersonSlug }) => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoadingPerson, setIsLoadingPerson] = useState(false);
+  const [isHasError, setIsHasError] = useState(false);
+
+  const isSelected = (person: Person) => person.slug === selectedPersonSlug;
+
+  const findPersonParents = (personParent: string | null) => {
+    return people.find(person => personParent === person.name);
+  };
+
+  useEffect(() => {
+    (async () => {
+      try {
+        setIsLoadingPerson(true);
+        const todosData = await getPeople();
+
+        setPeople(todosData);
+      } catch (error) {
+        setIsHasError(true);
+      } finally {
+        setIsLoadingPerson(false);
+      }
+    })();
+  }, []);
+
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {isLoadingPerson && <Loader />}
+
+        {isHasError && (
+          <p data-cy="peopleLoadingError" className="has-text-danger">
+            Something went wrong
+          </p>
+        )}
+
+        {!isLoadingPerson && !people.length && (
+          <p data-cy="noPeopleMessage">
+            There are no people on the server
+          </p>
+        )}
+
+        {!isLoadingPerson && people.length && (
```

Comment:
<img width="603" alt="Screenshot 2023-03-06 at 18 24 20" src="https://user-images.githubusercontent.com/53228259/223170114-c4f95fd6-d65e-4208-8803-0ee9f85e07cc.png">

so that 0 doesn't appear, it is necessary to use
```suggestion
        {!isLoadingPerson && people.length ? ( ... ) : null}
```